lib/taskana-core/src/test/java/acceptance/task/UpdateManualPriorityWithSpiAccTest.java

```diff
@@ -0,0 +1,147 @@
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.api.models.ObjectReference;
+import pro.taskana.task.api.models.Task;
+import pro.taskana.task.internal.builder.TaskBuilder;
+import pro.taskana.task.internal.jobs.helper.TaskUpdatePriorityWorker;
+import pro.taskana.workbasket.api.WorkbasketPermission;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.api.models.WorkbasketSummary;
+import pro.taskana.workbasket.internal.builder.WorkbasketAccessItemBuilder;
+
+@TaskanaIntegrationTest
+@WithServiceProvider(
+    serviceProviderInterface = PriorityServiceProvider.class,
+    serviceProviders = TestStaticValuePriorityServiceProvider.class)
+class UpdateManualPriorityWithSpiAccTest {
+
+  @TaskanaInject TaskanaEngine taskanaEngine;
+  @TaskanaInject TaskService taskService;
+  @TaskanaInject ClassificationService classificationService;
+  @TaskanaInject WorkbasketService workbasketService;
+
+  ClassificationSummary defaultClassificationSummary;
+  WorkbasketSummary defaultWorkbasketSummary;
+  ObjectReference defaultObjectReference;
+  TaskUpdatePriorityWorker worker;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    worker = new TaskUpdatePriorityWorker(taskanaEngine);
+    defaultClassificationSummary =
+        defaultTestClassification().buildAndStoreAsSummary(classificationService);
+    defaultWorkbasketSummary = defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .permission(WorkbasketPermission.APPEND)
+        .buildAndStore(workbasketService);
+    defaultObjectReference = defaultTestObjectReference().build();
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_setPriorityToManualAndNotUpdateAccordingToSpi_When_ManualPriorityPositive()
+      throws Exception {
+    Task task = taskService.newTask(defaultWorkbasketSummary.getId());
+    task.setClassificationKey(defaultClassificationSummary.getKey());
+    task.setPrimaryObjRef(defaultObjectReference);
+    task.setManualPriority(123);
+    taskService.createTask(task);
+    worker.executeBatch(List.of(task.getId()));
+    Task result = taskService.getTask(task.getId());
+
+    assertThat(result.getPriority()).isEqualTo(result.getManualPriority()).isEqualTo(123);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_setPriorityUsingSpi_When_ManualPriorityNegative() throws Exception {
+    Task task = taskService.newTask(defaultWorkbasketSummary.getId());
+    task.setClassificationKey(defaultClassificationSummary.getKey());
+    task.setPrimaryObjRef(defaultObjectReference);
+    task.setManualPriority(-1);
+    taskService.createTask(task);
+    Task result = taskService.getTask(task.getId());
+
```

Comment:
Same here. Move the space 1 up :) Fetching the task for verification purposes is not in the `when` state anymore.
If you disagree: Let's talk ^^